### PR TITLE
Person identifier cleaning

### PR DESCRIPTION
**Pull request by symroe**
This PR combines and replaces #2400 and #2406 into one PR.

I combined them because both PRs contained the same management command with different code in it. I decided to re-write it to make it more generic. 

I also did some cleaning up of the validation and cleaning code.

Tested on a staging copy of the database.

### CODE REVIEW

File: ynr/apps/people/tests/test_person_form_identifier_crud.py [hunk 1]
```diff
@@ -209,6 +266,24 @@ def test_mastodon_full_url(self):
         qs = PersonIdentifier.objects.all()
         self.assertEqual(qs[0].value, "joe")
 
+    def test_linkedin_urls(self):
+        urls_to_valid = (
+            ("http://example.com/@blah", False),
+            ("https://www.linkedin.com/in/first-last-57338a4/", True),
+        )
+        for url, valid in urls_to_valid:
```
[the remark this answers is not in the document]
Comment (symroe) [reply ynr/apps/people/tests/test_person_form_identifier_crud.py:274]: Unsure, but I don't think I can use `@pytest.mark.parametrize` in a UnitTest class? Hence the use of `subtest`

[the remark this answers is not in the document]
Comment (chris48s) [reply ynr/apps/people/tests/test_person_form_identifier_crud.py:274]: Oh right yeah.. gotcha. Ignore that then.

Covering the extra variants still stands.

File: ynr/apps/people/helpers.py [hunk 2]
```diff
@@ -120,6 +120,53 @@ def clean_twitter_username(username):
     return username
 
 
+def clean_linkedin_url(url):
+    parsed_url = urlparse(url)
+    valid = True
+    if parsed_url.netloc not in (
+        "linkedin.com",
+        "www.linkedin.com",
+        "uk.linkedin.com",
```
[the remark this answers is not in the document]
Comment (symroe) [reply ynr/apps/people/helpers.py:129]: Not in the actual data we have on record, but you're right. I've moved to a regex match on any subdomain here.

File: ynr/apps/people/tests/test_person_form_identifier_crud.py [hunk 3]
```diff
@@ -163,7 +162,30 @@ def _submit_mastodon_values(self, value, value_type="mastodon_username"):
         form["source"] = "They changed their username"
         form["tmp_person_identifiers-0-value_type"] = value_type
         form["tmp_person_identifiers-0-value"] = value
-        form.submit()
+        return form.submit()
+
+    def _submit_mastodon_values(self, value, value_type="mastodon_username"):
```
[the remark this answers is not in the document]
Comment (symroe) [reply ynr/apps/people/tests/test_person_form_identifier_crud.py:167]: Huh. I guess this was a previous rebase gone wrong. Removed

**Comment by symroe** [comment]
Ok, I reckon that's all of those comments addresses?

**Review by chris48s — APPROVED** [review]
Latest changes look pretty sensible. Ta

**Pull request merged**